Index: src/main/java/snaked/model/Difficulty.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package snaked.model;\r\n\r\nimport lombok.Getter;\r\n\r\npublic enum Difficulty {\r\n        EASY(1,1),\r\n        MEDIUM(2,2),\r\n        HARD(4,4);\r\n        @Getter private final double scoreMultiplier;\r\n        @Getter private final double speedMultiplier;\r\n\r\n        Difficulty(double scoreMultiplier, double speedMultiplier) {\r\n            this.scoreMultiplier = scoreMultiplier;\r\n            this.speedMultiplier = speedMultiplier;\r\n        }\r\n    }\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/snaked/model/Difficulty.java b/src/main/java/snaked/model/Difficulty.java
--- a/src/main/java/snaked/model/Difficulty.java	(revision 7b55afb84efe8e90b5592bd8a35f9b14bdff1e11)
+++ b/src/main/java/snaked/model/Difficulty.java	(date 1669727322478)
@@ -3,14 +3,16 @@
 import lombok.Getter;
 
 public enum Difficulty {
-        EASY(1,1),
-        MEDIUM(2,2),
-        HARD(4,4);
-        @Getter private final double scoreMultiplier;
-        @Getter private final double speedMultiplier;
+    EASY(1, 1),
+    MEDIUM(2, 2),
+    HARD(4, 4);
+    @Getter
+    private final double scoreMultiplier;
+    @Getter
+    private final double speedMultiplier;
 
-        Difficulty(double scoreMultiplier, double speedMultiplier) {
-            this.scoreMultiplier = scoreMultiplier;
-            this.speedMultiplier = speedMultiplier;
-        }
-    }
+    Difficulty(double scoreMultiplier, double speedMultiplier) {
+        this.scoreMultiplier = scoreMultiplier;
+        this.speedMultiplier = speedMultiplier;
+    }
+}
Index: src/main/java/snaked/model/Direction.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package snaked.model;\r\n\r\npublic enum Direction {\r\n    UP,\r\n    DOWN,\r\n    LEFT,\r\n    RIGHT;\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/snaked/model/Direction.java b/src/main/java/snaked/model/Direction.java
--- a/src/main/java/snaked/model/Direction.java	(revision 7b55afb84efe8e90b5592bd8a35f9b14bdff1e11)
+++ b/src/main/java/snaked/model/Direction.java	(date 1669727322462)
@@ -4,5 +4,5 @@
     UP,
     DOWN,
     LEFT,
-    RIGHT;
+    RIGHT
 }
Index: src/main/java/snaked/model/SnakeSkin.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package snaked.model;\r\n\r\nimport lombok.Getter;\r\n\r\nimport java.nio.file.Path;\r\n\r\npublic enum SnakeSkin {\r\n    //as soon as we have the images for the snake skin, we can add them as arguments for the enums\r\n    // ONE,\r\n    //TWO,\r\n    //THREE;\r\n;\r\n    @Getter private final Path bodyFileName;\r\n    @Getter private final Path headFileName;\r\n\r\n    private SnakeSkin(Path headFileName, Path bodyFileName){\r\n        this.headFileName=headFileName;\r\n        this.bodyFileName=bodyFileName;\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/snaked/model/SnakeSkin.java b/src/main/java/snaked/model/SnakeSkin.java
--- a/src/main/java/snaked/model/SnakeSkin.java	(revision 7b55afb84efe8e90b5592bd8a35f9b14bdff1e11)
+++ b/src/main/java/snaked/model/SnakeSkin.java	(date 1669727322429)
@@ -9,12 +9,14 @@
     // ONE,
     //TWO,
     //THREE;
-;
-    @Getter private final Path bodyFileName;
-    @Getter private final Path headFileName;
+    ;
+    @Getter
+    private final Path bodyFileName;
+    @Getter
+    private final Path headFileName;
 
-    private SnakeSkin(Path headFileName, Path bodyFileName){
-        this.headFileName=headFileName;
-        this.bodyFileName=bodyFileName;
+    SnakeSkin(Path headFileName, Path bodyFileName) {
+        this.headFileName = headFileName;
+        this.bodyFileName = bodyFileName;
     }
 }
Index: src/main/java/snaked/model/ExampleLombokClass.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package snaked.model;\r\n\r\nimport lombok.*;\r\n\r\n@RequiredArgsConstructor\r\n@EqualsAndHashCode(onlyExplicitlyIncluded=true)\r\n@ToString\r\npublic class ExampleLombokClass {\r\n    private final @Getter @EqualsAndHashCode.Include int id;\r\n    private @Getter @Setter String name;\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/snaked/model/ExampleLombokClass.java b/src/main/java/snaked/model/ExampleLombokClass.java
--- a/src/main/java/snaked/model/ExampleLombokClass.java	(revision 7b55afb84efe8e90b5592bd8a35f9b14bdff1e11)
+++ b/src/main/java/snaked/model/ExampleLombokClass.java	(date 1669727322445)
@@ -3,9 +3,11 @@
 import lombok.*;
 
 @RequiredArgsConstructor
-@EqualsAndHashCode(onlyExplicitlyIncluded=true)
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
 @ToString
 public class ExampleLombokClass {
-    private final @Getter @EqualsAndHashCode.Include int id;
-    private @Getter @Setter String name;
+    private final @Getter
+    @EqualsAndHashCode.Include int id;
+    private @Getter
+    @Setter String name;
 }
Index: src/main/java/snaked/model/BoardCell.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package snaked.model;\r\n\r\nimport javafx.scene.image.Image;\r\n\r\npublic enum BoardCell {\r\n    CONSUMABLE,\r\n    SNAKEPART,\r\n    WALL;\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/snaked/model/BoardCell.java b/src/main/java/snaked/model/BoardCell.java
--- a/src/main/java/snaked/model/BoardCell.java	(revision 7b55afb84efe8e90b5592bd8a35f9b14bdff1e11)
+++ b/src/main/java/snaked/model/BoardCell.java	(date 1669727322417)
@@ -1,9 +1,7 @@
 package snaked.model;
 
-import javafx.scene.image.Image;
-
 public enum BoardCell {
     CONSUMABLE,
     SNAKEPART,
-    WALL;
+    WALL
 }
\ No newline at end of file
