Index: src/main/java/snaked/model/GameControls.java
===================================================================
diff --git a/src/main/java/snaked/model/GameControls.java b/src/main/java/snaked/model/GameControls.java
deleted file mode 100644
--- a/src/main/java/snaked/model/GameControls.java	(revision 86e58c3b6b7c1f3c9820e21716df6242a8197c95)
+++ /dev/null	(revision 86e58c3b6b7c1f3c9820e21716df6242a8197c95)
@@ -1,35 +0,0 @@
-package snaked.model;
-
-import lombok.Getter;
-import lombok.Setter;
-
-public class GameControls {
-    private Snake snake;
-
-    @Setter private Direction currentDirection;
-    private GameBoard gameBoard;
-
-    private GameOptions gameOptions;
-
-    public GameControls(Snake snake, GameBoard gameBoard, GameOptions gameOptions){
-        this.snake = snake;
-        this.gameBoard = gameBoard;
-        this.currentDirection = gameOptions.getStartingDirection();
-        this.gameOptions = gameOptions;
-    }
-
-    public boolean checkAlive(){
-        return true;
-        //TODO: complete this method
-    }
-
-    public boolean nextTurn(){
-        gameBoard.updateSnakeCoordinates(currentDirection, false); // TODO: update ateConsumable
-        
-        return checkAlive();
-    }
-    public int getScore (){
-        return snake.getEatenConsumables()*gameOptions.getDifficulty().getScoreMultiplier();
-    }
-
-}
